lib/Mojo/Server/Daemon.pm

```diff
@@ -212,6 +212,12 @@ sub _listen {
   say 'Web application available at ', $options->{path} // $url;
 }
 
+sub _peer {
+  my $handle = shift;
+  my $peer   = $handle->isa('IO::Socket::UNIX') ? 'peerpath' : 'peerhost';
+  return $handle->$peer;
```

Comment:
Why do you insist on making it so verbose?
```perl
sub _peer { $_[0]->->isa('IO::Socket::UNIX') ? $_[0]->peerpath : $_[0]->peerhost }
```

Reply:
:+1:  Much better.  Updated.